packages/client-sdk-nodejs/src/internal/storage-data-client.ts

```diff
@@ -189,10 +198,10 @@ export class StorageDataClient implements IStorageDataClient {
     });
   }
 
-  public async put(
```

Comment:
`put` is not sufficient in javascript because when a user passes us a `number` we can't tell for sure whether they intended to store it as a double or as an integer.